pkg/artifactory/resource/configuration/resource_artifactory_backup_test.go

```diff
@@ -141,7 +142,7 @@ func TestAccCronExpressions(t *testing.T) {
 }
 
 func cronTestCase(cronExpression string, t *testing.T) (*testing.T, resource.TestCase) {
-	resourceName := fmt.Sprintf("artifactory_backup.backuptest")
+	fqrn := fmt.Sprintf("artifactory_backup.backuptest")
```

Comment:
One little thing... why use `Sprintf` when you don't have format here?

Reply:
you're right. For no reason. Replaced with `Sprint`

Reply:
Hmmm, why even use `fmt`? It's just a simple string assignment. `fqrn := "artifactory_backup.backuptest"` no?

Reply:
Omg, really :) fixed!